docusaurus/src/remark/utils.js

```diff
@@ -16,19 +16,52 @@ const isDocsPage = (vfile) => {
 };
 
 const getRegistryEntry = async (vfile) => {
-    const pathParts = vfile.path.split("/");
-    const connectorName = pathParts.pop().split(".")[0];
-    const connectorType = pathParts.pop();
-    const dockerRepository = `airbyte/${connectorType.replace(
-      /s$/,
-      ""
-    )}-${connectorName}`;
-
-    const registry = await catalog;
-
-    return registry.find(
-      (r) => r.dockerRepository_oss === dockerRepository
+  const pathParts = vfile.path.split("/");
+  const connectorName = pathParts.pop().split(".")[0];
+  const connectorType = pathParts.pop();
+  const dockerRepository = `airbyte/${connectorType.replace(
+    /s$/,
+    ""
+  )}-${connectorName}`;
```

Comment:
curious why this expression is multiline

Reply:
🤷 it's what my linter did